projects/igniteui-angular/src/lib/services/overlay/overlay.ts

```diff
@@ -977,4 +983,14 @@ export class IgxOverlayService implements OnDestroy {
             info.closeAnimationPlayer.finish();
         }
     }
+
+    private addComponentSize(info: OverlayInfo) {
+        if (info?.elementRef?.nativeElement && info.elementRef.nativeElement instanceof Element) {
```

Comment:
The info really always exists looking from where this is called, so no need to check and can be shortened to:
```suggestion
        if (info.elementRef?.nativeElement instanceof Element) {
```